odin/api/v1/core/views.py

```
@@ -72,3 +80,25 @@ def get(self, request: Request, *args: list, **kwargs: dict) -> HttpResponse:
+    authentication_classes = []
+    permission_classes = (AllowAny,)
+
+    def get(self, request: Request, *args, **kwargs):
+        if (context := get_cached_index_context()) is None:
+            context = update_index_context_cache()
+        serializer = DashboardSerializer(context)
+        return Response(serializer.data)
+
+
+class WeatherChartView(APIView):
+    authentication_classes = []
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)